augur/data/schema-export-v2.json

```
@@ -169,7 +169,8 @@
                         "properties": {
                             "key": {
                                 "description": "They key used to access the value of this coloring on each node",
-                                "type": "string"
+                                "type": "string",
+                                "not": {"enum": ["none"]}
```

```suggestion
                               "not": {"const": "none"}
```

is another way to spell this, but I guess we might want to disallow other specific values in the future…

---

Updated to `const` in 7736ca3ba4486ee6a917aa6000a6551a7f29bd82